bot.py

```diff
@@ -121,7 +121,7 @@ def run_script(self):
             ### BATTLE START
             self.image_processor.check_and_click_until_found(
                 self.template_images["TIME_LIMIT_INDICATOR"],
-                'Time limit indicator',
+                "Time limit indicator",
```

Comment:
Related to this issue:
https://github.com/AdriaGual/pokemon-pocket-bot/issues/4

It should be single quotes.

```suggestion
                'Time limit indicator',
```

Reply:
ok, sorry. by default pattern my python linter makes all double quotes. I will fix. also have another fix